packages/tinacms/src/admin/pages/CollectionListPage.tsx

```diff
@@ -631,267 +631,278 @@ const CollectionListPage = () => {
+                                                </span>
+                                                <span className="h-5 leading-5 block truncate">
+                                                  <span>
+                                                    {document.node.name}
+                                                  </span>
+                                                </span>
+                                              </span>
+                                            </a>
+                                          </td>
+                                          <td className="px-3 py-3" colSpan={4}>
+                                            <span className="block text-xs text-gray-400 mb-1 uppercase">
+                                              Path
+                                            </span>
+                                            <span className="leading-5 block text-sm font-medium text-gray-900 truncate">
+                                              {document.node.path
+                                                .substring(2)
+                                                .split('/')
+                                                .map((node) => {
+                                                  return (
+                                                    <span key={node}>
+                                                      <span className="text-gray-300 pr-0.5">
+                                                        /
+                                                      </span>
+                                                      <span className="pr-0.5">
+                                                        {node}
+                                                      </span>
+                                                    </span>
+                                                  )
+                                                })}
+                                            </span>
+                                          </td>
+                                        </tr>
+                                      )
+                                    }
+
+                                    const hasTitle = Boolean(
+                                      document.node._sys.title
+                                    )
+                                    const subfolders =
+                                      document.node._sys.breadcrumbs
+                                        .slice(0, -1)
+                                        .join('/')
+
                                     return (
-                                      <tr key={`folder-${document.node.path}`}>
-                                        <td className="pl-5 pr-3 py-3 truncate max-w-0">
+                                      <tr
+                                        key={`document-${document.node._sys.relativePath}`}
+                                        className=""
```

Comment:
**question**: Should the className attribute be empty?

Reply:
Good catch

Reply:
✅ Removed